### ISSUE TITLE
Action buttons are not consistent between AI and Controls page

### ISSUE THREAD

**Issue by sshveta** [konveyor/tackle-ui#64]
I think this is already known issue,
Edit and Delete action are present under Kebab menu in application Inventory page
but It is present as buttons on Controls page

![Screenshot from 2021-03-24 15-05-14](https://user-images.githubusercontent.com/3866976/112389677-5ef34280-8cb2-11eb-95fd-26a98e337e2d.png)

**Comment by mrizzi** [comment]
@sshveta that's right and, IIRC, this has been done because this page is going to become full of information and the final version will have no space for the `Edit` and `Delete` buttons we have in the Controls' pages.

So the idea basically has been about implementing already the "ultimate" version for Edit and Delete so that the users will always see them in a consistent way in next versions/releases.
